05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,36 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.raspberry {
+  width: 500px;
+  float: left;
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+.action {
+  width: 500px;
+  height: 394.28px;
+  background-color: rgb(240,240,240);
+  float: left;
```

Comment:
if both raspberry and action have the same width and float, can you factor them out?